### PR TITLE
changed favicon and title

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added favicon
2. changed title of the app

### CODE REVIEW

**Review by devdeadviz — APPROVED** [review]
The code is clean and your file structure is also good. Everything looks good to me!

**Review by sharmayatendra — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicons/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
```
Comment (xii-vi) [inline public/index.html:10]: You can add description too.
Just a suggestion.

**Review by xii-vi — APPROVED** [review]
Clean and organized code.
Left a sanity comment.

**Review by viru2001 — APPROVED** [review]

**Review by Rishi05051997 — COMMENTED** [review]
Ok Good to merge

**Pull request merged**